voxel51/users/api.py

```diff
@@ -185,6 +186,45 @@ def query_analytics(self, analytics_query):
         _validate_response(res)
         return _parse_json_response(res)
 
+    def get_analytic_id(self, name, version=None):
+        '''Gets the ID of the analytic with the given name (and optional
+        version).
+
+        Under the hood, this method uses an
+        :class:`voxel51.users.query.AnalyticsQuery` to search for the analytic
+        on the Platform. Thus, technically, `name` and `version` can be
+        substrings of the full analytic name and version. However, this method
+        raises an error if multiple analytics are found to match the search.
```

Comment:
This also raises an error if an analytic's name is a substring of another's - e.g. I have `foo` and `foo-bar`, so I can't search for `foo`.

Reply:
yeah good point. I better add explicit logic to always allow exact matches